### PR TITLE
Task 2 complete (full)

### PR DESCRIPTION
**Pull request by SkymeFactor**
Везь день дебажил, оказалось, датасет не загрузил xD Accuracy ~0.65, двухслойная нейронная сеть

### CODE REVIEW

**Comment by SkymeFactor** [comment]
Сделал задание 2.2, получил acc=0,81, но я не сильно оптимизировал, буквально пару раз запускал, хочу в график уложиться, и да, торч странный, ручками было веселее :)

File: Task2/SkymeFactor/layers.py [hunk 1]
```diff
@@ -34,9 +35,29 @@ def softmax_with_cross_entropy(preds, target_index):
...
+      probs = probs[0] + 1e-10
+      ce = - np.mean(np.log(probs)[target_index])
+
+    loss = ce #ross_entropy_loss(softmax(preds), target_index)
+    grad = probs
+    grad -= zeros
+
+    if preds.ndim > 1:
+      grad /= preds.shape[0]
+    return loss.mean(), grad
```
Comment (toshiks) [inline Task2/SkymeFactor/layers.py]: Очень хочется чтобы здесь использовались написанные тобой выше функции

Comment (SkymeFactor) [reply Task2/SkymeFactor/layers.py]: Просто изначально в шаблоне не было этих функций, я подумал, что нужно их реализовать как бы внутри кросс-энтропии, собственно, это то же самое, только склеенное в одну функцию
